content/references/gamification/ranks.md

```diff
@@ -0,0 +1,36 @@
+---
+kind: reference
+sidebar: ranks-honor
+prev: /references/gamification/
```

Comment:
There's nothing under this link. Should it lead to explanations/privileges?

Reply:
I guess, to be consistent with "the other way around" (I thought I had put htat one correctly... :/ )

```suggestion
prev: /explainations/gamification/privileges
```